src/lxc/tools/lxc_unshare.c

```diff
@@ -373,20 +428,19 @@ int main(int argc, char *argv[])
 			}
 
 			if (wait_for_pid(pid) != 0)
-				fprintf(stderr, "Could not move interface %s "
-						"into container %d: %s\n",
-					tmpif->mi_ifname, pid, strerror(errno));
+				SYSERROR("Could not move interface \"%s\" into container %d",
+				         tmpif->mi_ifname, pid);
 		}
 	}
 
-	if (daemonize)
+	if (my_args.daemonize)
 		exit(EXIT_SUCCESS);
 
 	if (wait_for_pid(pid) != 0) {
-		fprintf(stderr, "Failed to wait for '%d'\n", pid);
+		SYSERROR("Failed to wait for '%d'", pid);
 		exit(EXIT_FAILURE);
 	}
 
 	/* Call exit() directly on this function because it retuns an exit code. */
 	exit(EXIT_SUCCESS);
-}
+}
```

Comment:
Ah, there needs to be a newline at the end of the file. Otherwise it's (by POSIX standards) not a file. :)

Reply:
@brauner Thanks for your review :)